.github/workflows/spellcheck.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         id: spell
         with:
           dictionary: components/dictionary.txt
-          files: "docs/* docs/*/*"
+          files: "docs/* docs/*/* workshop_materials/cheatsheets/* instructor_notes/* instructor_notes/openscpca/* "
```

Comment:
I'd probably just remove this at this point and check everything. I'm not too worried about the partial words added being a problem.

I also thought of another thing we could  if we want to exclude particular files: add a step with `rm` for the files we want the spellchecker to skip.